app/src/lib/git/remote.ts

```diff
@@ -45,7 +45,7 @@ export async function removeRemote(
   name: string
 ): Promise<void> {
   const options = {
-    successExitCodes: new Set([0, 128]),
+    successExitCodes: new Set([0, 2]),
```

Comment:
Required because of: https://github.com/git/git/commit/9144ba4cf52bb0e891d7c10a331fc32c1d3e8f64#diff-d1216797fe2c919ca196cd3f78ac01f8119b1f5f04df11c0cd11011ee753c857R840